weave-js/src/components/Tag/Tag.tsx

```diff
@@ -86,7 +94,7 @@ export const RemovableTag: FC<RemovableTagProps> = ({
 }) => {
   const labelRef = useRef<HTMLParagraphElement>(null);
   const isTooltipEnabled = isTagLabelTruncated(labelRef);
-  const classes = useTagClasses({color});
+  const classes = useTagClasses({color, isInteractive: true});
```

Comment:
Should RemovableTagProps Omit 'isInteractive'?

Reply:
good call, will do!